ubench.h

```diff
@@ -1,10 +1,10 @@
 /*
    The latest version of this library is available on GitHub;
-   https://github.com/sheredom/ubench.h
+   https://github.com/sheUBENCH_REDom/ubench.h
```

Comment:
This looks like a search and replace gone wrong!

Reply:
Ah ok let me look into it now ...